analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SymbolHelperTest.cs

```diff
@@ -398,5 +398,185 @@ public void GetClassification_Method(MethodKind methodKind, string expected)
 
             fakeSymbol.Object.GetClassification().Should().Be(expected);
         }
+
+        [DataTestMethod]
+        [DataRow("BaseClass`1", "VirtualMethod", "MyInheritedAttribute", "MyDerivedInheritedAttribute", "MyNotInheritedAttribute", "MyDerivedNotInheritedAttribute", "MyUnannotatedAttribute")]
```

Comment:
Can we make the `BaseClass` longer, so it aligns with the rest? Like `BaseClassForUsages`, it will make it way more obvious

Reply:
oh, whitespace, neat! :)